libs/domains/variables/feature/src/lib/create-update-variable-modal/create-update-variable-modal.tsx

```diff
@@ -87,6 +88,7 @@ export function CreateUpdateVariableModal(props: CreateUpdateVariableModalProps)
       value: variable?.value ?? '',
       isSecret: variable?.is_secret,
       description: variable?.description,
+      enable_interpolation_in_file: variable?.enable_interpolation_in_file,
```

Comment:
You miss one point of the spec
![2024-08-20-180914_600x239_scrot](https://github.com/user-attachments/assets/2edaee5a-15a2-44f1-8fca-140087ccdc56)

```suggestion
      enable_interpolation_in_file: variable?.enable_interpolation_in_file ?? true,
```

Reply:
Fixed, thanks https://github.com/Qovery/console/pull/1576/commits/d3b06aecf7173f3033bd39614c68bfb71d4a21a0